pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -38,7 +38,11 @@ const (
 // +kubebuilder:printcolumn:name="Paused",type="boolean",JSONPath=".status.paused",description="Whether the resource reconciliation is paused or not",priority=1
 // +kubebuilder:subresource:status
 
-// ThanosRuler defines a ThanosRuler deployment.
+// The `ThanosRuler` custom resource definition (CRD) declaratively defines a desired [Thanos Ruler](https://github.com/thanos-io/thanos/blob/main/docs/components/rule.md) setup to run in a Kubernetes cluster. With Thanos Ruler recording and alerting rules can be processed across multiple Prometheus instances.
+//
+// A `ThanosRuler` instance requires at least one query endpoint which points to the location of Thanos Queriers or Prometheus instances.
+//
+// Further information can also be found in the [Thanos section]({{< ref "thanos.md" >}}).
```

Comment:
Do we know if this link will continue to work when moving things around?

Reply:
I think the link on line 45 will not work because originally this section was in Design.md and Thanos.md were in the same directory. I will look into this.

Reply:
Yes I would remove the link to avoid trouble.